mizar/dp/mizar/operators/endpoints/endpoints_operator.py

```diff
@@ -194,7 +194,8 @@ def delete_endpoints_from_bouncers(self, bouncer):
     def delete_bouncer_from_endpoints(self, bouncer):
         eps = self.store.get_eps_in_net(bouncer.net).values()
         for ep in eps:
-            ep.update_bouncers(set([bouncer]), False)
+            if ep.type == OBJ_DEFAULTS.ep_type_simple or ep.type == OBJ_DEFAULTS.ep_type_host:
```

Comment:
How about other EP types, do they need to update bouncer? My understanding is that for scaled endpoint and gateway endpoint we need to update bouncers too. Did I miss something?

Reply:
Resolve it based on the discussion with Phu and Cathy.